### PR TITLE
Add settings for dask cluster

### PR DESCRIPTION
**Pull request by jsignell**
For use in dask cluster creation on setup  CU-daq3f8

### CODE REVIEW

**Comment by hhuuggoo** [comment]
Task linked: [CU-daq3f8 Define dask cluster for example projects](https://app.clickup.com/t/daq3f8)

**Comment by jsignell** [comment]
This is ready. Not sure if we want one for examples-gpu

**Review by jameslamb — COMMENTED** [review]
Love it! Should the GPU examples get a Dask cluster by default too? I have no opinion, but maybe @rikturr or @skirmer does

**Comment by jsignell** [comment]
> Should the GPU examples get a Dask cluster by default too

Same page lol. I pulled it from https://nbviewer.jupyter.org/github/saturncloud/examples/blob/main/examples/examples-gpu/nyc-taxi/rf-rapids-dask.ipynb#Initialize-Dask-GPU-cluster so should be good

File: .ci/validate-examples.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 FILENAME_REGEX = r"^[0-9A-Za-z\-\.]+$"
 SATURN_DIR_NAME = ".saturn"
 SATURN_JSON_NAME = "saturn.json"
-SATURN_JSON_KEYS = ["image", "jupyter", "environment_variables"]
+SATURN_JSON_KEYS = ["image", "jupyter", "environment_variables", "description"]
```
Comment (jsignell) [inline .ci/validate-examples.py:27]: This is unrelated, but I noticed it and think this is better.

Comment (jameslamb) [reply .ci/validate-examples.py:27]: oh yeah works for me!

**Review by jameslamb — APPROVED** [review]

**Comment by jsignell** [comment]
woohoo!!

**Comment by jsignell** [comment]
Actually... ping @saturncloud/data-science for visibility :)

**Pull request merged**